pkg/cmd/codespace/jupyter.go

```diff
@@ -0,0 +1,82 @@
+package codespace
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/cli/cli/v2/pkg/liveshare"
+	"github.com/spf13/cobra"
+)
+
+func newJupyterCmd(app *App) *cobra.Command {
+	var codespace string
+
+	jupyterCmd := &cobra.Command{
+		Use:   "jupyter",
```

Comment:
Yeah I think that `gh cs open` makes a ton of sense as the general-purpose/scalable solution. Though for a set of high-pri scenarios, it probably also makes sense having an alias to simplify their opening experience? Since Jupyter is ubiquitous in data science/ML (like VS Code is with web/etc. audiences), it feels meaningful to have a top-level alias for it.

Reply:
Makes sense! Yeah as long as we are thinking strategic about it so the API doesn't bloat (though flags aren't THAT much better)

Reply:
https://github.com/cli/cli/discussions/5496